src/pages/state/[state].js

```diff
@@ -114,6 +114,9 @@ export default function State({ data: { statesGeoData }, state = "DC" }) {
   stateData.notes = notes?.filter(({ tags }) =>
     tags.includes(stateData.abbreviation)
   );
+  stateData.voicesForReform = stateData.notes.filter(
+    note => note.voicesForReform
+  );
```

Comment:
Thinking since we last spoke: how does the boolean interact with OPO tags? Looks like we're just ignoring them and using the state abbreviation, which is fine, but worth confirming/noting in the admin CMS docs

Reply:
Also with this logic would the note be repeated in both the Voices for Reform section _and_ News and Notes section?